test/rules/std_L009_L052_test.py

```diff
@@ -0,0 +1,23 @@
+"""Tests the python routines within L009 and L052."""
+from sqlfluff.core import FluffConfig
+from sqlfluff.core import Linter
+
+
+def test__rules__std_L009_and_L052_interaction() -> None:
```

Comment:
Nitpick: Could we avoid creating a custom test here by creating a test in `test/fixtures/linter/autofix/` instead? Those tests can run multiple rules.

Reply:
@barrywhart I'm struggling with this a bit. I can see that the autofix `test-config.yml` accepts multiple rules, but this test also requires a custom config setting to enable final semi-colons and there aren't any other examples of this in the autofix tests. 
tried similar to the std rule test syntax but that is unrecognised and raises an error. 
Can you let me know the syntax or if these tests even allow the custom config option?

I'll take a look in the code myself, but if it's not there then we can stick with custom test.

Reply:
okay it's not possible to add a custom config to these tests so lets leave the custom python test for now.
![image](https://user-images.githubusercontent.com/80432516/143680896-956f0147-d4f6-42be-85fb-b0a3d6a0a4ab.png)

If we want to add more detailed yaml config to these tests then that can be a separate PR 👍